vllm/entrypoints/openai/cli_args.py

```diff
@@ -77,7 +77,7 @@ def __call__(
 def make_arg_parser(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
     parser.add_argument("--host",
                         type=nullable_str,
-                        default=None,
+                        default="0.0.0.0",
```

Comment:
IMHO this should be set to `127.0.0.1`. However, this would be a breaking change and should be changed carefully.

Reply:
Thank you for the consideration not breaking compatibility!